probman.tex

```diff
@@ -0,0 +1,11 @@
+\section{Data Management Problem/Conflict Resolution }
+The above organisational structure allocates significant responsibility to lead institutions.  As such, when problems arise that cannot be solved with the responsibility and scope allocated to an institution, the path of escalation and resolution of such problems must be clear. 
+
+Any intra institutional problem should be bought as early as possible to the DM Project manager. The PM will attempt to mediate a resolution. The PM will consult with DMLT, DM System Science Team and DM System engineering if there are Scientific or technical impacts to be considered. 
```

Comment:
do we mean "inter" here instead? "Intra-institutional" means "inside the institution" doesn't it?